apps/src/gamelab/AnimationTab/NewListItem.jsx

```diff
@@ -62,7 +62,7 @@ var NewListItem = React.createClass({
   render: function () {
     var hovered = Radium.getState(this.state, 'main', ':hover');
     return (
-      <div style={styles.tile} onClick={this.props.onClick}>
+      <div style={styles.tile} onClick={this.props.onClick} id="newListItem">
```

Comment:
There can be more than one `NewListItem` on the page (in fact there is right now - one in the 'animations' column and one in the 'frames' column).  Should either pass an id to this component and make it unique, or give it a className instead.